src/coffee/client.coffee

```diff
@@ -220,9 +226,11 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    if options.enableRepeater? and !options.enableRepeater
+      @_task = new TaskQueue
```

Comment:
Question: do we really need to keep 2 task queues? Can't we just have e.g: one which can optionally work as repeater but by default is a "normal" task queue?